src/app.js

```diff
@@ -217,6 +217,21 @@ This instance of Remix you are visiting WILL NOT BE UPDATED.\n
 Please make a backup of your contracts and start using http://remix.ethereum.org`)
   }
 
+  function importExternal (url, cb) {
+    handleImports.import(url,
+      (loadingMsg) => {
+        $('#output').append($('<div/>').append($('<pre/>').text(loadingMsg)))
```

Comment:
can't we do this without jQuery?
1. it would be nicer if we do not grab randomly a node from the dom (`#output`) but have some kind of event or api
2. then, the place that controls the `#output` node, should use regular dom methods `.appendChild(...)` rather than jQuery

Reply:
+1 for getting rid of jQuery :)

Reply:
that lead to an interesting improvement.
replaced jquery by `tooltip` but then the tooltip would pop up at each compilation (basically for every key stroke), so added a cache so we don't display the popup if already imported.

Reply:
and also we don't anymore make the request to retrieve the content at each compilation (that's the interesting improvement)